TestAssets/TestProjects/LightupClient/project.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Yes.  And it still uses the CLI build framework 😉 

There is an effort to move this repo to MSBuild.  See https://github.com/dotnet/core-setup/tree/buildtools